reference/constraints.rst

```diff
@@ -50,6 +50,7 @@ Validation Constraints Reference
    constraints/Currency
    constraints/Luhn
    constraints/Iban
+   constraints/Bic
```

Comment:
I'm sorry, but I can't find such file. Did you mean `reference/constraints/map.rst.inc` instead?